### PR TITLE
Fail gracefully by returning null if there was an error in attempting to read from registry

### PR DESCRIPTION
**Pull request by meenakb**

### CODE REVIEW

**Comment by MSLaguana** [comment]
I suggest that we also improve taco.ts to check for errors as well; have the promise chain in the main `run()` function end with a done() so we don't silently swallow errors

File: src/taco-cli/cli/taco.ts [hunk 1]
```diff
@@ -107,12 +107,12 @@ class Taco {
                             });
```
Comment (MSLaguana) [inline src/taco-cli/cli/taco.ts]: We'll also have to make sure to return the Taco.runWithParsedArgs promise above now

**Comment by meenakb** [comment]
I agree, improving taco.ts to error out in case of any error in the entire promise chain.

**Comment by meenakb** [comment]
Comments addressed.

**Comment by MSLaguana** [comment]
Looks good to me now

**Pull request merged**